contracts/modules/assessment/Assessment.sol

```diff
@@ -62,6 +62,9 @@ contract Assessment is IAssessment, MasterAwareV2 {
     config.payoutCooldownInDays = 1; // days
     config.stakeLockupPeriodInDays = 14; // days
     config.silentEndingPeriodInDays = 1; // days
+
+    // to receive NXM
+    ITokenController(getInternalContractAddress(ID.TC)).addToWhitelist(address(this));
```

Comment:
Because Assessment has the stake() unstake() operations which involve the Assessment contract holding NXM.

This was not caught at unit testing because the NXM token mock does not implement a whitelist.

Reply:
good catch. was thinking about rewards only, but you're right.